src/bin/latency.rs

```diff
@@ -138,6 +149,7 @@ fn do_server(iface_name: String, oneway: bool) {
         },
         false => None,
     };
+    let mut rx_map: HashMap<usize, SystemTime> = HashMap::new();
```

Comment:
The name `rx_map` is ambiguous. And, we don't need to use it. Don't think about packet shuffling. Just save *last id* and *last rx timestamp*.

1. Got sync packet and last id matches → Print elapsed time
2. Got Tx packet that id < last id → Packet shuffled. But very rare. Just drop it to recv matching Sync packet
3. Got Sync packet that id < last id → Ignore it too
4. Got Sync packet that id > last id → We lost Tx packet. As we didn't received matching Tx packet, Print error (Not panic)